.circleci/config.yml

```diff
@@ -404,6 +404,76 @@ jobs:
           name: Run tests
           command: yarn test -w 1
 
+  test-e2e:
+    docker:
+      - image: mcr.microsoft.com/playwright:focal
+        auth:
+          username: $DOCKER_USER
+          password: $DOCKER_PASS
+        environment:
+          DJANGO_SETTINGS_MODULE: marsha.settings
+          PYTHONPATH: /home/circleci/marsha/src/backend
+          DJANGO_CONFIGURATION: Test
+          DJANGO_SECRET_KEY: ThisIsAnExampleKeyForTestPurposeOnly
+          DJANGO_JWT_SIGNING_KEY: ThisIsAnExampleKeyForDevPurposeOnly
+          POSTGRES_HOST: localhost
+          POSTGRES_DB: marsha
+          POSTGRES_USER: marsha_user
+          POSTGRES_PASSWORD: pass
+          POSTGRES_PORT: 5432
+          DJANGO_AWS_ACCESS_KEY_ID: aws-access-key-id
+          DJANGO_AWS_SECRET_ACCESS_KEY: aws-secret-access-key
+          DJANGO_CLOUDFRONT_DOMAIN: abc.cloudfront.net
+          DJANGO_UPDATE_STATE_SHARED_SECRETS: dummy,secret
+          DJANGO_AWS_MEDIALIVE_ROLE_ARN: aws:medialive:arn:role
+          DJANGO_AWS_MEDIAPACKAGE_HARVEST_JOB_ARN: aws:mediapackage:arn:role
+          DOCKERIZE_VERSION: v0.6.1
+          DJANGO_ALLOW_ASYNC_UNSAFE: 1
+      - image: circleci/postgres:10.3-alpine-ram
+        auth:
+          username: $DOCKER_USER
+          password: $DOCKER_PASS
+        environment:
+          POSTGRES_DB: marsha
+          POSTGRES_USER: marsha_user
+          POSTGRES_PASSWORD: pass
+    working_directory: ~/marsha/src/backend
+    parameters:
+      browser:
+        description: "Playwright browser"
+        default: "firefox"
+        type: string
+    steps:
+      - checkout:
+          path: ~/marsha
+      - run:
+          name: Install e2e dependencies
+          command: pip install --user .[dev,e2e]
+      - run:
+          name: Install front-end dependencies
+          working_directory: ~/marsha/src/frontend
+          command: yarn install --frozen-lockfile
+      - run:
+          name: Build front-end application
+          working_directory: ~/marsha/src/frontend
+          command: yarn build
```

Comment:
I don't get what you mean.
Since playwright requires a specific image base, we have to rebuild everything in it.
Can you explain your proposal, please ?

Reply:
```suggestion
          command: yarn build --mode=production
```